src/lib/importer/kelzFormatParser.tsx

```diff
@@ -323,15 +235,21 @@ type Affixes = {
   step: number
 }
 
-function readRelicStats(relic, part, grade, enhance) {
-  let mainStat
-  if (part === 'Hands') {
-    mainStat = Constants.Stats.ATK
-  } else if (part === 'Head') {
-    mainStat = Constants.Stats.HP
-  } else {
-    mainStat = mapMainStatToId(relic.mainstat)
-  }
+function readRelicStats(relic: V4ParserRelic, part: string, grade: number, enhance: number, scanner: KelzFormatParser) {
+  const mainStat = ((relic: V4ParserRelic, part: string) => {
+    switch (part) {
+      case 'Hands':
+        return Constants.Stats.ATK
+      case 'Head':
+        return Constants.Stats.HP
+      default:
+        return mapMainStatToId(relic.mainstat)
+    }
+  })(relic, part)
```

Comment:
I changed this one along with the generic args, trying to avoid this pattern of defining a function then calling it immediately, should be a separate function